package.json

```diff
@@ -1,5 +1,6 @@
 {
-  "name": "@openmrs/esm-patient-management",
+  "name": "@icap-ethiopia/esm-patient-management",
```

Comment:
@sinteacon it seems changes from ur fork sneaked into this PR. Please review the files and revert these changes

Reply:
sorry for the messed up, I just reverted the changes.